backgroundJobs/priceFetcher.js

```diff
@@ -55,6 +55,86 @@ const specialAssetPriceFetchers = {
     //CAN (cannon)
     return "0.0000000000000001"
   },
+  BSC_0xd4CB328A82bDf5f03eB737f37Fa6B370aef3e888: async (web3, network, stakedTokenAddress) => {
```

Comment:
why don't you call the general price function here?
the one that calls coingecko.

Reply:
as discussed this is a different API